File: OnlineStore/OnlineStoreUI/src/model/entities/apiRequests/productDataRequest.ts
export class ProductDataRequest{
    idEntity:number;
    price: number;
    status: string;
    idProductInformation: number;

    constructor(idEntity:number, price: number, status: string, idProductInformation: number){
        this.idEntity = idEntity;
        this.price = price;
        this.status = status;
        this.idProductInformation = idProductInformation;
    }
}

File: OnlineStore/Api/BLContracts/Models/ProductDataRequest.cs
using System.Collections.Generic;

namespace BLContracts.Models
{
	public class ProductDataRequest
	{
		public int IdEntity { get; set; }
		public decimal Price { get; set; }
		public string Status { get; set; }
		public int IdProductInformation { get; set; }

	}
}
